easybuild/easyconfigs/o/OpenKIM-API/OpenKIM-API-1.9.2-foss-2016b.eb

```diff
@@ -0,0 +1,30 @@
+name = 'OpenKIM-API'
+version = '1.9.2'
+easyblock = 'ConfigureMake'
```

Comment:
@schiotz Style remark: please put the `easyblock` line on top, include empty line below (for consistency with existing easyconfigs).